dbscripts/xml/upgrade/3.0.0_review_method.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 3.0.0_review_method.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 3.0.0.0 review_method update.
+  -->
+<data>
+	<sql><!-- #2195 set review_method = 2 (double-blind) where review_method = 0 i.e. for those migrated from OJS 2.4.x -->
+		<query>UPDATE review_assignments SET review_method = 2 WHERE review_method = 0</query>
```

Comment:
Suggest adding the PHP constants in a comment so that greping for them will turn up these magic numbers too.